frontend/packages/knative-plugin/src/topology/sidebar/KnativeOverviewSections.tsx

```diff
@@ -0,0 +1,105 @@
+import { RevisionModel } from '../../models';
+import { usePodsForRevisions } from '../../utils/usePodsForRevisions';
+import { isServerlessFunction } from '../knative-topology-utils';
+import { Subscriber } from '../topology-types';
+
+type KnativeOverviewDetailsProps = {
+  item?: OverviewItem;
+};
+
+type OverviewListProps = {
+  items: K8sResourceKind[];
+};
+
+type SubscriptionsOverviewList = {
+  subscriptions: Subscriber[];
+};
+
+export const EventSourcesOverviewList: React.FC<OverviewListProps> = ({ items }) => {
+  const { t } = useTranslation();
+  return items?.length > 0 ? (
+    <TopologySideBarTabSection>
+      <PubSubResourceOverviewList items={items} title={t('knative-plugin~Event Sources')} />
+    </TopologySideBarTabSection>
+  ) : null;
+};
+
+export const SubscriptionsOverviewList: React.FC<SubscriptionsOverviewList> = ({
+  subscriptions,
+}) => {
+  const { t } = useTranslation();
+  return subscriptions?.length > 0 ? (
+    <TopologySideBarTabSection>
+      <EventPubSubSubscribers
+        subscribers={subscriptions}
+        title={t('knative-plugin~Subscriptions')}
+      />
+    </TopologySideBarTabSection>
+  ) : null;
+};
+
+export const TriggersOverviewList: React.FC<SubscriptionsOverviewList> = ({ subscriptions }) => {
+  const { t } = useTranslation();
+  return subscriptions?.length > 0 ? (
+    <TopologySideBarTabSection>
+      <EventPubSubSubscribers subscribers={subscriptions} title={t('knative-plugin~Triggers')} />
+    </TopologySideBarTabSection>
+  ) : null;
+};
+
+export const DomainMappingsOverviewList: React.FC<OverviewListProps> = ({ items }) => {
+  const { t } = useTranslation();
+  return items?.length > 0 ? (
+    <TopologySideBarTabSection>
+      <DomainMappingOverviewList
+        domainMappings={items}
+        title={t('knative-plugin~Domain mappings')}
+      />
+    </TopologySideBarTabSection>
+  ) : null;
+};
+
+export const KnativeOverviewRevisionPodsRing: React.FC<KnativeOverviewDetailsProps> = ({
+  item,
+}) => {
+  const { obj } = item;
+  const { pods } = usePodsForRevisions(obj.metadata.uid, obj.metadata.namespace);
+  return (
+    <div className="resource-overview__pod-counts">
+      <PodRing
+        pods={pods?.[0]?.pods || []}
+        obj={obj}
+        rc={pods?.[0]?.obj}
+        resourceKind={RevisionModel}
+        path="/spec/replicas"
+      />
+    </div>
+  );
+};
+
+export const KnativeOverviewDetails: React.FC<KnativeOverviewDetailsProps> = ({ item }) => {
+  const { obj } = item;
+  return (
+    <div className="overview__sidebar-pane-body resource-overview__body">
+      {obj.kind === RevisionModel.kind ? <KnativeOverviewRevisionPodsRing item={item} /> : null}
+      <div className="resource-overview__summary">
+        <ResourceSummary resource={obj} />
+      </div>
+      {isServerlessFunction(obj) && (
+        <div className="resource-overview__details">
+          <ServerlessFunctionType />
+        </div>
+      )}
+    </div>
+  );
+};
```

Comment:
this is repeating here https://github.com/openshift/console/blob/master/frontend/packages/knative-plugin/src/components/overview/KnativeOverview.tsx#L13-L44

Reply:
Yes,it's still used in `KnativeResourceOverviewPage.tsx` which will be removed when all the knative sidepanels are migrated then we can remove the `KnativeOverview.tsx` file as well

Reply:
with this PR coming in, they are not in use so do you think we can remove it now?